developer-docs-site/docs/tutorials/validator-node/using-source-code.md

```diff
@@ -9,7 +9,7 @@ sidebar_position: 13
 1. Clone the Aptos repo.
 
       ```
-      git clone https://github.com/<YOUR-GITHUB-USERID>/aptos-core
```

Comment:
We should revisit other docs and evaluate recommendations we're making. It might not be a great idea asking folks that don't understand git to fork from our branch.